### ISSUE TITLE
ww3_ounf compiled with NCO switch for fixed file has a problem on the turn of the year

### ISSUE THREAD

**Issue by juanliria** [NOAA-EMC/WW3#293]
The NCO switch for ww3_ounf contains a bug that overwrites the ww3.gridded.nc file on the turn of year.
So, on a forecast that goes from the 26th of december 2020 to the 2nd of January 2021. ww3_ounf will write a ww3.gridded.nc file for 2020 and then overwrite the same file with the 2021 data, as a result there will be a single ww3.gridded.nc file containing data from 2021 and the 2020 data will be lost.

Basically does the same as without the NCO switch but instead of creating two files ww3.2020.nc and ww3.2021.nc overwrites the existing ww3.gridded.nc with the 2021 data (this will happen again next year).
